inc/class-wpseo-replace-vars.php

```diff
@@ -1142,14 +1142,14 @@ private function get_replacement_variables() {
 	}
 
 	/**
-	 * Checks if the replacement variable contains a prefix.
+	 * Checks whether the replacement variable contains a `ct_` or `cf_` prefix, because they follow different logic.
 	 *
 	 * @param string $replacement_variable The replacement variable.
 	 *
 	 * @return bool True when the replacement variable is not prefixed.
 	 */
 	private function is_not_prefixed( $replacement_variable ) {
```

Comment:
Why do you check whether something is *not* prefixed? Double negotiations are hard to read.